modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -78,6 +81,8 @@ GrainTracker::validParams()
 GrainTracker::GrainTracker(const InputParameters & parameters)
   : FeatureFloodCount(parameters),
     GrainTrackerInterface(),
+    _merge_grains_basedMisorAngle(getParam<bool>("merge_grains_basedMisorAngle")),
+    _euler(getUserObject<EulerAngleProvider>("euler_angle_provider")),
```

Comment:
This is one of the big reasons we don't want to modify this class. The GrainTracker works for toy Voronoi problems where we don't care about orientation. We don't want to enforce that every use of this class now have to consider Euler Angles. This requirement should be added to your new GrainTrackerMerge class.

Reply:
Yeah, I have fallen back to a version that didn't add this part of the functionality and tried implementing it in GrainTrackerMerge;